web/static/js/components/surveys/SurveyEdit.jsx

```diff
@@ -29,9 +29,8 @@ class SurveyEdit extends Component {
     const { dispatch, projectId, surveyId, router } = this.props
     if (projectId && surveyId) {
       projectActions.fetchProject(projectId)
-      dispatch(actions.fetch(projectId, surveyId))
+      dispatch(actions.fetchSurveyIfNeeded(projectId, surveyId))
```

Comment:
The first is just there to simplify the reading of the second.

It's being exported just in case that someone need to force the fetch for some reason. Maybe we can keep it private for now until there is a good reason for that.